src/status_im/multiaccounts/login/core.cljs

```diff
@@ -403,6 +403,7 @@
               (data-store.chats/fetch-chats-rpc
                {:on-success
                 #(do (re-frame/dispatch [:chats-list/load-success %])
+                     (rf/dispatch [:communities/get-user-requests-to-join])
```

Comment:
Discussed with @cammellos, currently the users requests are fetched on login. 
Should we just do this when the user first navigates to the communities home page? 

This list will also be re-fetched when a user cancels a request, or makes a new request.

Reply:
@J-Son89, I think the reasoning is that we only need to fetch once the app initializes to get the initial & latest state from status-go. Later on, the backend will push signals to the mobile client. And all the mobile client has to do is reconcile the new state with the existing app db state.

This is a rather efficient process. So as long as the signals are coming and the mobile logic to process them is correct, things should just work. But are the signals for community requests flowing to the client?

Reply:
Btw, by currently I'm by comment above. I mean, as of this pr this is how I implemented it. 
This call to get the lost of users requests was not in place before this pr

Reply:
I see now. I shared more what would be the ideal scenario IMO, where the mobile client only needs to react to changes coming from the DB after the initialization, hence the client would not make ad-hoc RPC calls to get the most up-to-date state from the backend because this is bug-driven.

Reply:
@cammellos do we have some form of pub sub for stuff like this?

I would not really consider it ad-hoc. They are just standard lifecycle checks imo. Basically when user logs in and whenever there is a request to the backend to adjust the `users-lists-of-requests`. 
Perhaps I should check if the cancelRequest method comes with a response (ideally the updated list) and then I can remove an unnecessary call to get the list again, as I have it now.
Same for making a request. 

@borismelnik - do you know if there is a response from cancelling/ creating a request to join a community. 

Ideally we could generate some docs for Status Go.

Reply:
@J-Son89 about docs - I totally agreed :)
If I understand you correct, you can find it in `MessageResponse.RequestsToJoinCommunity`. Im a little bit out of current discussion, but I did it in exactly the same way as usual request to join. So when you cancel some request to community, technically you just send it like request to join with special status. So processing the result should be in the same way as it was with requests to join

Reply:
> @J-Son89, I think the reasoning is that we only need to fetch once the app initializes to get the initial & latest state from status-go. Later on, the backend will push signals to the mobile client. And all the mobile client has to do is reconcile the new state with the existing app db state.

I believe that's the correct/desired behavior, so:

1) As the user logs in, we fetch all (or paginated if there's pagination) requests to join (this could be done on component load not to have to store everything in memory from the get go, but it's the same conceptually).
2) Status go will signal new requests through `messages.new`, this are the requests that come from the network (syncing maybe?).  This request should be reconciled with the app-db state.
3) When you create a request, you hit the RPC to create a request in status-go, and status-go will return (not through a signal, but through the callback response) the newly created request, which again will be reconciled

now, that's the theory, I'll have a quick look at the code to see if that's the case.